examples/analytics.amp.html

```diff
@@ -442,6 +442,39 @@
 </script>
 </amp-analytics>
 <!-- End SOASTA mPulse example -->
+
+<!-- Adobe Analytics example -->
+<!--
+For complete documentation and additional examples please see: https://marketing.adobe.com/resources/help/en_US/sc/implement/accelerated-mobile-pages.html
+-->
+<amp-analytics type=adobeanalytics>
+<script type="application/json">
+{
+  "requests": {
+    "myClick": "${click}&v1=${eVar1}"
+  },
+  "vars": {
+    "host": "metrics.example.com",
+    "reportSuites": "reportSuiteID"
+  },
+  "triggers": {
+    "pageLoad": {
+      "on": "visible",
+      "request": "pageview"
+    },
+    "click": {
+      "on": "click",
+      "selector": "#test1",
+      "request": "myClick",
+      "vars": {
+        "eVar1": "button clicked"
+      }
+    }
+  }
+}
+</script>
+</amp-analytics>
+
```

Comment:
can you add an example for adobeanalytics_nativeconfig as well? The part that I do not understand is where the data is going. The config does not mention any requests or url to load.